client/state/data-layer/wpcom/sites/comments/likes/mine/delete/test/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_UNLIKE,
+	COMMENTS_LIKE,
+	NOTICE_CREATE,
+} from 'state/action-types';
+import {
+	unlikeComment,
+	updateCommentLikes,
+	handleUnlikeFailure,
+} from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const SITE_ID = 91750058;
+const POST_ID = 287;
+
+describe( '#unlikeComment()', () => {
+	const action = {
+		type: COMMENTS_UNLIKE,
+		siteId: SITE_ID,
+		postId: POST_ID,
+		commentId: 1
+	};
+
+	it( 'should dispatch a http action to remove a comment like', () => {
+		const dispatch = spy();
+		unlikeComment( { dispatch }, action );
+
+		expect( dispatch ).to.have.been.calledOnce;
+		expect( dispatch ).to.have.been.calledWith( http( {
+			apiVersion: '1.1',
+			method: 'POST',
+			path: `/sites/${ SITE_ID }/comments/1/likes/mine/delete`,
+			onSuccess: action,
+			onFailure: action,
+			onProgress: action,
+		} ) );
+	} );
+} );
+
+describe( '#updateCommentLikes()', () => {
+	it( 'should dispatch a comment like update action', () => {
+		const dispatch = spy();
+
+		updateCommentLikes( { dispatch }, { siteId: SITE_ID, postId: POST_ID, commentId: 1 }, null, { i_like: true, like_count: 4 } );
+
+		expect( dispatch ).to.have.been.calledOnce;
+		expect( dispatch ).to.have.been.calledWith( {
+			type: COMMENTS_LIKE,
+			siteId: SITE_ID,
+			postId: POST_ID,
+			commentId: 1,
+			iLike: true,
```

Comment:
on an earlier note I don't think we need to carry the `iLike` here because it seems already captured in the action type. if we wanted to send a like with `iLike: false` then I think it would be more appropriate to send `COMMENTS_UNLIKE`